recipes/quilt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "quilt" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quiltdata/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ded69d039949de8293b0c98e95cd0a958eb7e68c93106b1f35254dd4691685c4
+
+build:
+  number: 0
+  skip: true  # [py2k]
+  script: 
+      - cd api/python
+      - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - appdirs >=1.4.0
+    - aws-requests-auth >=0.4.2
+    - boto3 >=1.8.0 
+    - elasticsearch 6.3.1
+    - jsonlines 1.2.0
+    - numpy >=1.14.0               # required by pandas, but missing from its dependencies.
```

Comment:
``numpy`` is not missing from ``pandas`` dependency

Reply:
https://github.com/conda-forge/pandas-feedstock/blob/6e5683fde2b01d8e8344459da6f8944451f77347/recipe/meta.yaml#L28